### ISSUE TITLE
Improvement for astrophoto (normal lens, no telescope)

### ISSUE THREAD

**Issue by Flore-mozfr** [Stellarium/stellarium#219]
Hello,

First, thank you for this great software that I've been using for years.

I am planning my next astrophoto session (lunar eclipse), I wanted to use stellarium to simulate the picture. I am using a DSLR with a wide angle regular lens.
I was able to enter the CCD data for my camera (no problem), but then I got stuck, because I could not (virtually in stellarium) directly put the lens on the camera. I always have to select a telescope. And I don't know how to create a "neutral" telescope (meaning a 1x).
I already took pictures with this camera/lens so I have a good idea of what this should look. I just want to make sure I can include everything I want on the picture.

Basically I expect the same rectangle but with data from the lens.

So if anyone has an idea on how to do that or wants to add that feature, it would be great.

**Comment by gzotti** [comment]
Your lens has a focal length. Add a "telescope" with these data.

**Comment by Flore-mozfr** [comment]
OK, I was so focused on "lens", that I did not see it as a telescope. But it makes sense.
It works of course.
Thank you. This can be closed
